io/disk/iozone.py

```diff
@@ -448,7 +448,7 @@ def setUp(self):
                         self.cancel('btrfs-tools is needed for the test to be run')
 
         tarball = self.fetch_asset(
-            'http://www.iozone.org/src/current/iozone3_434.tar')
+            'http://www.iozone.org/src/current/iozone3_492.tar')
```

Comment:
@maramsmurthy we have 493 as latest version. any reason for adding one level below the latest available?
BTW, please add this as yaml parameter, so that we can use easily pass this from yaml itself instead of changing each time in the code. let me know if u need any help here

Reply:
Latest tar file is iozone3_492.tar, iozone3_492.tar is a tgz file. So I tried using tar as no need to do any changes to the existing.
As you said we can try moving iozone version to yaml file. Will do this change and push the code again.